pkgs/hfsevents.nix

```diff
@@ -1,19 +0,0 @@
-{ mkDerivation, base, bytestring, cereal, Cocoa, CoreServices
```

Comment:
Did you meant to delete this?

Reply:
No 😅 there was some mixup with the stack2nix.patch application. I’ll add another commit fixing it momentarily. 

Once I sort out the cabal2nix issue this sort of junk shouldn’t come up 👍